packages/map-popup/src/index.tsx

```diff
@@ -114,9 +115,7 @@ export function MapPopup({ closePopup = () => {}, configCompanies, entity, getEn
 
   const bikesAvailablePresent = entityIsStation(entity)
   const entityIsStationHub = bikesAvailablePresent && entity?.bikesAvailable !== undefined && !entity?.isFloatingBike;
-  const stopId = !bikesAvailablePresent && entity?.code || entity.id.split(":")[1] || entity.id
-
-  // Double quotes make the query invalid, so remove them from the id just in case
+  const stopId = !bikesAvailablePresent && entity?.code;  // Double quotes make the query invalid, so remove them from the id just in case
```

Comment:
You can remove this comment

```suggestion
  const stopId = !bikesAvailablePresent && entity?.code;
```